README.md

```diff
@@ -2,135 +2,137 @@
 
 ![madness](https://cloud.githubusercontent.com/assets/8445924/10410421/ccca8b24-6f30-11e5-9952-9e5be5c4d792.png)
 
-Manul is the madness dependencies vendoring utility for Golang programs.
+Manul is the dependency vendoring utility for Golang programs.
```

Comment:
In my opinion, calling your product `madness` is not a good choice. However, if you insist, I will revert edit.